renderdoc/driver/gl/gl_initstate.cpp

```diff
@@ -484,10 +498,14 @@ bool GLResourceManager::Prepare_InitialState(GLResource res)
   }
   else if(res.Namespace == eResFramebuffer)
   {
-    byte *data = AllocAlignedBuffer(sizeof(FramebufferInitialData));
-    RDCEraseMem(data, sizeof(FramebufferInitialData));
+    if(!GetInitialContents(Id).blob)
```

Comment:
Maybe `HasInitialContents`, or add operator == and test against `InitialContentData()` ?